src/main/scala/io/gitlab/agluszak/tickets/http/responses/ReservationCreateResponse.scala

```diff
@@ -0,0 +1,31 @@
+package io.gitlab.agluszak.tickets.http.responses
+
+import java.time.LocalDateTime
+
+import io.gitlab.agluszak.tickets.{Reservation, ScreeningId, TicketTypeId}
+
+sealed trait ReservationCreateResponse extends Product with Serializable
```

Comment:
Is `extends Product with Serializable` neccessary? Scala adds it automatically for `case class` anyway